### DISCUSSION TITLE
How does superset worker ensure that every task is executed only once？

### DISCUSSION THREAD

**Discussion by LuPan2015** [apache/superset#18700] [Q&A / Help]
In superset beat, the ID of the scheduled task that meets the conditions will be written to redis. If there are multiple workers at the same time, how to ensure that each task is executed only once?

**Comment by zhaoyongjie** [comment]
Hi @LuPan2015, GitHub issue list is used to track bugs now, so I transfer this topic to the discussion board.

**Comment by LuPan2015** [comment]
thanks, I have learned
